challenges/primes.js

```diff
@@ -10,3 +10,21 @@
 */
 
 // YOUR CODE HERE
+
+var p =[];
+function primes(max) {
+     for (var i = 2; i < max ; i++) {
+	  if (isPrime(i) === true){
```

Comment:
Anytime you're checking if something `=== true`, you can just check that thing! You could just say `if(isPrime(i))`.